.github/ISSUE_TEMPLATE/kubernetes_update.md

```diff
@@ -22,7 +22,6 @@ List of items to do for upgrading to {version_1} {version_2} {version_3}
     - [ ] Minikube in `build/includes/minikube.mk` (Get the patch version [here](https://kubernetes.io/releases/) since minikube supports the latest Kubernetes release)
     - [ ] Kind in `build/includes/kind.mk` (Confirm {version_2} is supported and get the patch version [here](https://github.com/kubernetes-sigs/kind/releases))
 - [ ] Update the k8s image used in the helm [pre-delete-hook](https://github.com/googleforgames/agones/blob/main/install/helm/agones/templates/hooks/pre_delete_hook.yaml) to {version_2} (Get the patch version [here](https://hub.docker.com/r/lachlanevenson/k8s-kubectl))
-- [ ] Update client-go in `go.mod` and `test/terraform/go.mod` to {version_2} by running `go get k8s.io/client-go@{CORRESPONDING_VERSION}` and `go get k8s.io/apiextensions-apiserver@{CORRESPONDING_VERSION}`, then re-run `go mod tidy` and `go mod vendor`
```

Comment:
we still need this step for `go.mod`, right?

Reply:
Ah! Good catch. Will update, rather than delete 👍🏻

Reply:
This should be better now, and ready for re-review.